website/docs/docs/deploy/ci-jobs.md

```diff
@@ -29,6 +40,37 @@ dbt Labs recommends that you create your Slim CI job in a dedicated dbt Cloud [d
 
 3. In the **Triggers** section, choose the **Continuous Integration** (CI) tab. Then, enable the **Run on Pull Requests** option. This configures pull requests and new commits to be a trigger for the Slim CI job.
 
+</TabItem>
+
+<TabItem value="beta" label="Beta version">
+
+To make CI job creation easier, many options on the **CI job** page are set to default values that dbt Labs recommends that you use. You can change some of these defaults and there are some that you can’t (which are greyed out).
```

Comment:
need to follow up on "greyed out" UX

Reply:
Which options are these?

Also, might be useful to differentiate between disabled options (user can't edit) and read-only options (only uneditable if user doesn't have access or is not in edit mode).

Reply:
hey @thorn14 : all of them are listed and described in their relative sections in the UI: https://deploy-preview-3806--docs-getdbt-com.netlify.app/docs/deploy/ci-jobs?version=beta

Reply:
with the latest UI looks like all options can be changed. updated wording to align.